patches/common/chromium/.patches.yaml

```diff
@@ -273,6 +273,10 @@ patches:
   owners: deepak1556
   file: disable-recursive-surface-sync.patch
   description: null
+-
+  owners: brenca
+  file: disable-redraw-lock.patch
```

Comment:
Any reasons to add it in the middle of the patches list, and not at the end?

Reply:
I just tried to maintain alphabetical order :)